app/src/main/java/com/android/shelfLife/model/recipe/RecipeGeneratorOpenAIRepository.kt

```diff
@@ -32,193 +27,186 @@ import kotlinx.serialization.json.putJsonArray
 import kotlinx.serialization.json.putJsonObject
 
 @Singleton
-class RecipeGeneratorOpenAIRepository
-@Inject
-constructor(private val openai: OpenAI, @IoDispatcher private val dispatcher: CoroutineDispatcher) :
+class RecipeGeneratorOpenAIRepository @Inject constructor(private val openai: OpenAI) :
     RecipeGeneratorRepository {
 
   companion object {
-    const val USER_PROMPT_USE_TOOL_CALL =
-        "To ensure a great tastin recipe, you can add other ingredients you deem relevant and necessary. Call _createRecipeFunction with a list of ingredients (name + quantity), step by step instructions, servings, and cooking time in seconds"
-    const val USE_SOON_TO_EXPIRE_SYSTEM_PROMPT =
-        "You are an assistant who generates recipes using ingredients that are about to expire."
-    const val USE_SOON_TO_EXPIRE_USER_PROMPT =
-        "Create a recipe using the following ingredients that are about to expire: "
+    val QUICK_RECIPE_DEF = "quick, 15minutes"
+    val LONG_RECIPE_DEF = "long, over 30minutes"
```

Comment:
using 15 min seems kind of too specific. Wouldnt it be better to have something like "quick, under 15minutes" just like how you did for long?

Reply:
I updated this like you said